Lib/test/test_platform.py

```diff
@@ -430,6 +443,39 @@ def test_libc_ver(self):
         self.assertEqual(platform.libc_ver(filename, chunksize=chunksize),
                          ('glibc', '1.23.4'))
 
+    def test_android_ver(self):
+        res = platform.android_ver()
+        self.assertIsInstance(res, tuple)
+        self.assertEqual(res, (res.release, res.api_level,
+                               res.manufacturer, res.model, res.device))
+
+        if sys.platform == "android":
+            for name in ["release", "manufacturer", "model", "device"]:
+                with self.subTest(name):
+                    value = getattr(res, name)
+                    self.assertIsInstance(value, str)
+                    self.assertGreater(len(value), 0)
```

Comment:
I suggest to log the value when the test fails:

```suggestion
                    self.assertNotEqual(value, "")
```